lib/Net/Async/Blockchain/BTC.pm

```diff
@@ -95,7 +95,7 @@ async sub subscribe {
         $self->$curry::weak(
             sub {
                 my ($self, $message) = @_;
-                return $self->subscription_response($subscription, $message);
+                return $self->subscription_response($subscription, unpack('H*', $message));
```

Comment:
question: Curious to know why unpacking here? Since first the raw data will be received on low level `on_read_ready` sub of `ZMQ.pm` package, why not cleaning the data there itself and in Net::Async::Blockchain::BTC you will have a cleaned data already?

Reply:
@tahir-deriv this is to take advantage of the already existing ZMQ client for Tron network as well, since tron will not make use of this unpack there, is better to bring this responsibility up to the currency provider, so we can use the same client for both.